tests/integration/cloudformation/api/test_stacks.py

```diff
@@ -293,3 +294,160 @@ def stack_process_is_finished(cfn_client, stack_name):
         "PROGRESS"
         not in cfn_client.describe_stacks(StackName=stack_name)["Stacks"][0]["StackStatus"]
     )
+
+
+@pytest.mark.skip(reason="Not implemented")
+@pytest.mark.aws_verified
+def test_drift_detection_on_lambda(deploy_cfn_template, cfn_client, lambda_client, snapshot):
```

Comment:
just a mental note: Looking over this made me realize we probably should include this in our "resource testing framework" (i.e. include a test for this for each resource) when we start on the new providers.

Reply:
I moved this test into its own file so we remember that it should be populated.